keyboards/quokka/rules.mk

```diff
@@ -0,0 +1,10 @@
+EXTRAKEY_ENABLE = yes
+LTO_ENABLE = yes
+RGB_MATRIX_SUPPORTED = yes
+RGB_MATRIX_ENABLE = yes
+AUDIO_ENABLE = yes
+OLED_ENABLE = yes
```

Comment:
```suggestion
```
These parameters are already specified in `info.json` (except `RGB_MATRIX_SUPPORTED`, but nobody sets that to `yes` anyway).